make_railroad_diagram/mg_token_rules.txt

```diff
@@ -15,39 +15,61 @@ Diagram(
 // Number
 Diagram(
     Comment('Number'),
-    Choice(
-        0,
-        NonTerminal('Number_Base_10'),
-        Sequence(Terminal('0x'), NonTerminal('Number_Base_16')),
-        Sequence(Terminal('0o'), NonTerminal('Number_Base_8')),
-        Sequence(Terminal('0b'), NonTerminal('Number_Base_2')),
-        Sequence(Terminal('/[2-9]|[1-2][0-9]|3[0-6]/'), Terminal('r'), NonTerminal('Number_Base_N'))
+    Sequence(
+        Optional(Terminal('/[-+]/')),
+        Choice(
+            0,
+            NonTerminal('Number_Base_10'),
+            Sequence(Terminal('0x'), NonTerminal('Number_Base_16')),
+            Sequence(Terminal('0o'), NonTerminal('Number_Base_8')),
+            Sequence(Terminal('0b'), NonTerminal('Number_Base_2')),
+            Sequence(Terminal('0t'), NonTerminal('Number_Base_T')),

```

Comment:
Base T is listed here between Base 2 and Base N, but below is explained after Base N. Perhaps keep the order consistent in both places?

Reply:
I have adjusted the order.